hathifiles_database.gemspec

```diff
@@ -8,19 +8,19 @@ Gem::Specification.new do |spec|
   spec.authors = ["Bill Dueber"]
   spec.email = ["bill@dueber.com"]
 
-  spec.summary = %q{Keep a database of the data in the hathifiles}
+  spec.summary = "Keep a database of the data in the hathifiles"
   spec.homepage = "https://github.com/billdueber/hathifiles_database"
   spec.license = "MIT"
```

Comment:
Is it still a MIT license? LICENSE.txt seems to say otherwise.

Reply:
Modified BSD now? I can't ID them on sight. Anyway, homepage and license updated.